contributor-guide/modules/ROOT/pages/best-practices.adoc

```diff
@@ -1,4 +1,39 @@
 = Best Practices
 :navtitle: Best Practices
 
-*Content under construction*
+This section contains guidance on how to reduce development time from concept through to publishing of a new library.
+
+== Common Dependencies
```

Comment:
Maybe "usual dependencies"? By "common dependencies" I thought it would be about dependencies common to all libraries.